### DISCUSSION TITLE
Yourls without Cookies

### DISCUSSION THREAD

**Discussion by Friedinger** [YOURLS/YOURLS#3127] [Q&A] [answered]
Hi,
is there a way to use Yourls without any Cookies? I don't want the counting system, I just want to redirect and short my links.

**Comment by ozh** [comment, marked as answer]
Cookies are used as a login helper on the user's side to avoid entering login and password for each admin page viewing.

Hits (ie number of clicks) on short URLs are stored in the MySQL database.
